src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -35,7 +36,27 @@ public class RestoreCommand
 
         public Guid ParentId { get; }
 
-        public const string ProjectRestoreInformation = "ProjectRestoreInformation";
+        // names for ProjectRestoreInformation and intervals
+        private static readonly string ProjectRestoreInformation = nameof(ProjectRestoreInformation);
+        private static readonly string Errors = nameof(Errors);
+        private static readonly string ErrorMessages = nameof(ErrorMessages);
+        private static readonly string Warnings = nameof(Warnings);
+        private static readonly string WarningMessages = nameof(WarningMessages);
+        private static readonly string Success = nameof(Success);
+
+        // names for child events for ProjectRestoreInformation
+        private static readonly string ExecuteRestoreAsyncInformation = nameof(ExecuteRestoreAsyncInformation);
+        private static readonly string BuildAssetsFileInformation = nameof(BuildAssetsFileInformation);
+        private static readonly string ValidateRestoreGraphsAsyncInformation = nameof(ValidateRestoreGraphsAsyncInformation);
+        private static readonly string VerifyCompatibilityAsyncInformation = nameof(VerifyCompatibilityAsyncInformation);
+        private static readonly string CreateRestoreResultInformation = nameof(CreateRestoreResultInformation);
+        private static readonly string NoOpInformation = nameof(NoOpInformation);
+
+        // names for intervals in NoOpInformation
+        private static readonly string NoOpEvaluateCacheFileDuration = nameof(NoOpEvaluateCacheFileDuration);
+        private static readonly string NoOpSuccessEvaluationDuration = nameof(NoOpSuccessEvaluationDuration);
+        private static readonly string NoOpSuccessEvaluation = nameof(NoOpSuccessEvaluation);
```

Comment:
can we have more meaningful names which actually reflect that field and why that is being raised? i wasn't able to understand anything from `NoOpSuccessEvaluationDuration` or  `NoOpSuccessEvaluation`